packages/odyssey-react-mui/src/DataTable/DataTable.tsx

```diff
@@ -728,12 +735,23 @@ const DataTable = ({
       </ScrollableTableContainer>
 
       {hasPagination && (
-        <DataTablePagination
+        <Pagination
           pagination={pagination}
           setPagination={setPagination}
+          lastRow={lastRow}
           totalRows={totalRows}
           isDisabled={isEmpty}
           variant={paginationType}
+          labels={{
+            rowsPerPage: t("pagination.rowsperpage"),
+            page: t("pagination.page"),
+            previous: t("pagination.previous"),
+            next: t("pagination.next"),
+            loadMore: t("pagination.loadmore"),
+            total: totalRows
+              ? t("pagination.rowswithtotal", { firstRow, lastRow, totalRows })
+              : t("pagination.rowswithouttotal", { firstRow, lastRow }),
+          }}
```

Comment:
The should be memoized with `useMemo`.

Reply:
On second thought, why aren't these just props on the component? Is it easier to maintain this as an object?

I don't like sending objects down to components because they break React's HTML-like structure.

**General comment:** A React component is a function that takes an object. Here, we're passing a sub-object to it. But the memoization system and everything else are all designed around props.

So in general, I don't like passing objects or arrays unless there's a good reason. I don't see a reason for it here. Just add more props to the component itself, and it will be far easier to document them in Storybook. It's not like they're optional either (which would be like `func(arg1, arg2, options)`).

Reply:
They'll be named something like `rowsPerPageLabel`, `currentPageLabel`, etc.

Reply:
Done.